index.js

```diff
@@ -11,3 +12,112 @@ x.addEventListener('click', () => {
+  },
+  {
+    modalTitle: 'Facebook 360',
+    modalTitle2: 'FACEBOOK',
+    modalListTech: ['Full Stack Dev', '2015'],
+    modalList: ['html', 'css', 'javascript'],
+    modalImg: './assets/Snapshoot Portfolio.png',
+    modalText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    modalTitle: 'Uber Navigation Stories',
+    modalTitle2: 'Uber',
+    modalListTech: ['Lead Devveloper', '2018'],
+    modalList: ['html', 'css', 'javascript'],
+    modalImg: './assets/Snapshoot Portfolio (f).png',
+    modalText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+
+];
```

Comment:
Nice work using an array of objects to store the projects data :clap: :clap: However, the link to the source code and the link to the live demo are missing. I suggest adding them for each object.